book_source/03_intermediate_users_guide/01_pecan_xml.Rmd

```diff
@@ -165,20 +165,43 @@ Only if this section is defined an ensemble analysis is done.
 
 #### functions: 
 
-`write.configs()`, `run.ensemble.analysis()`
+`write.configs()`,`write.ensemble.configs`, `run.ensemble.analysis()`
 
 #### tags
 
 ```xml
 <ensemble>
 	<size>5</size>
-  <variable>GPP</variable>
-  <variable>NPP</variable>
+	<start.year>1995</start.year>
+  <end.year>1999</end.year>
+  <samplingspace>
+  <parameters>
+      <method>lhc</method>
+  </parameters>
+    
+  <met>
+      <method>sampling</method>
+  </met>
+    
+  <soil>
+     <method>sampling</method>    
+  </soil>
+    
+  <vegetation>
+    <method>sampling</method>
+    <parent>met</parent>
+  </vegetation>
+  
+  </samplingspace>
 </ensemble>
 ```
 
 * **size** : [required] the number of runs in the ensemble.
-* **variable**: [optional] name of one (or more) variables the analysis should be run for. If not specified, sensitivity.analysis variable is used, otherwise default is GPP.
```

Comment:
Why has the `variable` tag been removed? At the moment that still needs to be there for the postprocessing code